src/main/kotlin/org/jitsi/nlj/util/Util.kt

```diff
@@ -47,7 +47,7 @@ infix fun Int.floorMod(other: Int): Int {
     return Math.floorMod(this, other)
 }
 
-inline fun getStackTrace(): String = with(StringBuffer()) {
+fun getStackTrace(): String = with(StringBuffer()) {
```

Comment:
If this isn't inline, it will include the `getStackTrace` frame on the stack, which we don't want.

Reply:
Oh, I see.

(Not that this function seems to be used anywhere - can we just drop it?)

Reply:
I see JVB has its own (in ByteBufferBool) that we use for bookkeeping...this version would actually be better (because of the inlining).

Reply:
(Of course it should ideally be in `jitsi-utils-kotlin`)